cves/kernel/CVE-2019-9162.yml

```diff
@@ -387,37 +418,41 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      This vulnerability would have been fixed if the original module had thought
```

Comment:
This solution you present happens many times throughout the file. Sometimes it feels irrelevant to the context of certain questions and can be removed in places like these.

Reply:
Updated